src/test/java/seedu/address/logic/commands/people/PeopleAddCommandTest.java

```diff
@@ -22,6 +22,8 @@
 import seedu.address.model.ReadOnlyAddressBook;
 import seedu.address.model.ReadOnlyUserPrefs;
 import seedu.address.model.person.Person;
+import seedu.address.model.transaction.Expense;
+import seedu.address.model.transaction.Income;
 import seedu.address.testutil.PersonBuilder;
 
 public class PeopleAddCommandTest {
```

Comment:
People add for wallet commands?

Reply:
Cos there’s a model stub below that needs to implement some wallet-related methods